baal/active/active_loop.py

```diff
@@ -79,7 +79,10 @@ def step(self, pool=None) -> bool:
             indices = None
 
         if len(pool) > 0:
-            probs = self.get_probabilities(pool, **self.kwargs)
+            if self.heuristic.__class__.__name__ == "Random":
```

Comment:
I think we can use `isinstance(self.heuristic, Random)` here.

Reply:
Thanks for the comment! I added a unit test that checks whether `self.get_probabilities` is called with `unittest.mock.patch`.